### PR TITLE
ESP32-C6 Workshop review and changes in the layout

### LINKED ISSUE #79: Add link to the workshops and fix the ESP-IDF workshop issues
Add link to the workshops and fix the ESP-IDF workshop issues

### PR DESCRIPTION
**Pull request by pedrominatel**
This PR includes:

- Fix the workshop and include additional information - From the Summit feedback
- Change in the Hugo configuration for the list to basic
- Add the Workshop menu
- Set the WIP workshops as draft

Closes #79

### CODE REVIEW

**Comment by f-hollow** [comment]
Please remove a space in the file `ESP RainMaker_horizontal.svg` at this path:

- `content/workshops/iot-introduction-with-rainmaker/assets/ESP RainMaker_horizontal.svg`

File: hugo.toml [hunk 1]
```diff
@@ -1,9 +0,0 @@
-baseURL = "https://developer.espressif.com/"
```
[the remark this answers is not in the document]
Comment (pedrominatel) [reply hugo.toml]: Reverted.

**Comment by pedrominatel** [comment]
Thanks @f-hollow for your review. PTAL.

File: content/_index.md [hunk 2]
```diff
@@ -2,10 +2,12 @@
 title: "Welcome to Espressif Developer Portal! :tada:"
 description: "This page is the Espressif Developer Portal."
 disableComments : true
+showRecent : false
 
 cascade:
     showLikes : false
     showViews : false
+    showTableOfContents : false
```
[the remark this answers is not in the document]
Comment (f-hollow) [reply content/_index.md:10]: Then should we revert these changes for now?

File: content/workshops/esp-idf-with-esp32-c6/assignment-5/index.md [hunk 3]
```diff
@@ -29,7 +29,11 @@ Install the provisioning application on your smartphone.
 
 2. **Create a new project from the examples**
 
-Create a new ESP-IDF project using the example `provisioning` -> `wifi_prov_mgr`.
+{{< alert icon="circle-info">}}
+For this assignment, we will create a new project based on the example [wifi_prov_mgr](https://github.com/espressif/esp-idf/tree/master/examples/provisioning/wifi_prov_mgr).
+{{< /alert >}}
+
+Create a new ESP-IDF project using the example `provisioning` -> `wifi_prov_mgr` as starting point.
```
[the remark this answers is not in the document]
Comment (f-hollow) [reply content/workshops/esp-idf-with-esp32-c6/assignment-5/index.md:36]: Ah, sorry! This command is for adding components from the ESP Component Registry. When I saw this command, the combination of `idf.py` and `example` led me to believe this is to create a project based on an example from ESP-IDF application examples.

I will resolve this thread then.

**Review by f-hollow — APPROVED** [review]
One comment to be decided on. Otherwise, LGTM!

**Pull request merged**